.github/workflows/main.yml

```diff
@@ -15,11 +15,6 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-      # Merge master branch into pull request to check whether it would be successful after merge
```

Comment:
@erikhofer do you think this should be done in a PR CI job?

Reply:
yes probably